test/parallel/test-util-inspect.js

```diff
@@ -2564,47 +2564,184 @@ assert.strictEqual(
     inspect(thrower, { getters: true }),
     '{\n' +
     '  foo: [Getter: <Inspection threw (Error: Oops\n' +
-    '    at get foo (/foo/node_modules/foo.js:2:7)\n' +
-    '    at get bar (/foo/node_modules/bar.js:827:30))>]\n' +
+    '      at get foo (/foo/node_modules/foo.js:2:7)\n' +
+    '      at get bar (/foo/node_modules/bar.js:827:30))>]\n' +
     '}',
   );
 };
 
 // Property getter throwing an error with getters that throws.
 // https://github.com/nodejs/node/issues/60683
 {
-  const error = new Error();
+  const badError = new Error();
+
+  const innerError = new Error('Oops');
+  innerError.stack = [
+    'Error: Oops',
+    '    at get foo (/foo/node_modules/foo.js:2:7)',
+    '    at get bar (/foo/node_modules/bar.js:827:30)',
+  ].join('\n');
 
   const throwingGetter = {
     __proto__: null,
     get() {
-      throw error;
+      throw innerError;
     },
     configurable: true,
     enumerable: true,
   };
 
-  Object.defineProperties(error, {
+  Object.defineProperties(badError, {
     name: throwingGetter,
     message: throwingGetter,
     stack: throwingGetter,
     cause: throwingGetter,
   });
 
   const thrower = {
-    get foo() { throw error; }
+    get foo() { throw badError; }
   };
 
   assert.strictEqual(
     inspect(thrower, { getters: true }),
     '{\n' +
     '  foo: [Getter: <Inspection threw ([object Error] {\n' +
-    '  stack: [Getter/Setter],\n' +
-    '  name: [Getter],\n' +
-    '  message: [Getter],\n' +
-    '  cause: [Getter]\n' +
-    '})>]\n' +
-    '}',
+    '    stack: [Getter/Setter: <Inspection threw (Error: Oops\n' +
+    '        at get foo (/foo/node_modules/foo.js:2:7)\n' +
+    '        at get bar (/foo/node_modules/bar.js:827:30))>],\n' +
+    '    name: [Getter: <Inspection threw (Error: Oops\n' +
+    '        at get foo (/foo/node_modules/foo.js:2:7)\n' +
+    '        at get bar (/foo/node_modules/bar.js:827:30))>],\n' +
+    '    message: [Getter: <Inspection threw (Error: Oops\n' +
+    '        at get foo (/foo/node_modules/foo.js:2:7)\n' +
+    '        at get bar (/foo/node_modules/bar.js:827:30))>],\n' +
+    '    cause: [Getter: <Inspection threw (Error: Oops\n' +
+    '        at get foo (/foo/node_modules/foo.js:2:7)\n' +
+    '        at get bar (/foo/node_modules/bar.js:827:30))>]\n' +
+    '  })>]\n' +
+    '}'
+  );
+}
+
+// Property getter throwing an error with getters that throws recursivly.
+{
+  const terminalError = new Error('Oops');
+  terminalError.stack = [
+    'Error: Oops',
+    '    at get foo (/foo/node_modules/foo.js:2:7)',
+  ].join('\n');
+
+  function getRecursiveThrowingError(recursions) {
+    const badError = new Error();
+
+    const innerError = recursions > 0 ?
+      getRecursiveThrowingError(recursions - 1) :
+      terminalError;
```

Comment:
Actually, I would not want a terminal error, I'd like to test an infinite recursion to make sure we do not crash in this case (i.e. `inspect` has to give up at some point)